test/topology/test_change_ip.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test clusters can restart fine after an IP address change.
+"""
+
+import logging
+import pytest
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_change_two(manager, random_tables):
+    """Add a node, stop an original node, add a column"""
```

Comment:
The test doesn't add any new nodes. Which I think is good - no need to make the test longer.

Reply:
it was a copy-paste comment, fixed.